### PR TITLE
WL-3627. Making the mouse pointer look clickable

### PR DESCRIPTION
**Pull request by surya-janani**
css effects when hover over selected item

### CODE REVIEW

File: library/src/webapp/skin/neo-ox/portal.css [hunk 1]
```diff
@@ -861,12 +861,13 @@ ul#otherSiteList li a span, ul.otherSitesCategorList li a span, ul.otherQuickLin
...
+    cursor: pointer;
 }
-/* eliminate the normal hover effect when it is current tool */
+
 #toolMenu li.selectedTool a:hover {
-    background: inherit !important;
-    *background: #f3f3f3 !important;
+    /*background: inherit !important;
+    *background: #f3f3f3 !important;*/
+    color:white!important;
```
Comment (nicholaswilson100) [inline library/src/webapp/skin/neo-ox/portal.css]: Why are you adding commented out code?

File: reference-base.iml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_6" inherit-compiler-output="false">
+    <output url="file://$MODULE_DIR$/target/classes" />
+    <output-test url="file://$MODULE_DIR$/target/test-classes" />
+    <content url="file://$MODULE_DIR$">
+      <excludeFolder url="file://$MODULE_DIR$/target" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="library" scope="TEST" name="Maven: junit:junit:4.11" level="project" />
+    <orderEntry type="library" scope="TEST" name="Maven: org.hamcrest:hamcrest-core:1.3" level="project" />
```
Comment (nicholaswilson100) [inline reference-base.iml]: We don't include .imls in git

Comment (nicholaswilson100) [inline reference-base.iml]: i.e., so this file needs to be backed out of the request

**Comment by nicholaswilson100** [comment]
References to other requests?

**Comment by surya-janani** [comment]
I did git reset --hard and force pushed my changes to remove reference.iml .

Retaining css changes

File: library/src/webapp/skin/neo-ox/portal.css [hunk 3]
```diff
@@ -861,12 +861,11 @@ ul#otherSiteList li a span, ul.otherSitesCategorList li a span, ul.otherQuickLin
...
     background:none repeat scroll 0 0 #fff;
-    cursor: default;
+    cursor: pointer;
 }
 /* eliminate the normal hover effect when it is current tool */
 #toolMenu li.selectedTool a:hover {
-    background: inherit !important;
-    *background: #f3f3f3 !important;
+    color:white!important;
 }
```
Comment (nicholaswilson100) [inline library/src/webapp/skin/neo-ox/portal.css:869]: Doesn't this colour the text white when you hover over it?

Comment (surya-janani) [inline library/src/webapp/skin/neo-ox/portal.css:869]: yes. It makes text go white when we hover over it

Comment (nicholaswilson100) [inline library/src/webapp/skin/neo-ox/portal.css:869]: cool

**Pull request merged**